.github/workflows/merge.yaml

```diff
@@ -25,6 +25,8 @@ jobs:
 
       - name: Run Tests
         run: make test
+        env:
+          SHELL: /bin/bash
```

Comment:
Why is this env var needed for this unit test?

Reply:
When the tests are running and attempt to:
```
// GetContext spawns a new shell session and returns its context
func GetContext(verbose bool) *Context {
	context, err := SpawnShell(CreateGoExpectSpawner(), defaultTimeout, Verbose(verbose), SendTimeout(defaultTimeout))
	if err != nil || context == nil || context.GetExpecter() == nil {
		log.Panicf("can't get a proper context for test execution")
	}
	return context
}
```

It errors out saying it can't get a proper context because it doesn't know what SHELL environment to use.  We have to be explicit in the env running the test.

Reply:
Thanks!